xconfess-backend/src/config/database.config.ts

```
@@ -30,8 +30,12 @@ export const getTypeOrmConfig = (
     password: configService.get<string>('DB_PASSWORD'),
     database: configService.get<string>('DB_NAME'),
     entities: [__dirname + '/../**/*.entity{.ts,.js}'],
+
+    migrations: [__dirname + '/../../migrations/*{.ts,.js}'],
```

Biome (2.4.4)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)